### ISSUE TITLE
Multiple pkg tests started failing on Fedora

### ISSUE THREAD

**Issue by Ch3LL** [saltstack/salt-jenkins#496]
The following pkg tests are failing on fedora:

```
integration.states.test_pkg.PkgTest.test_pkg_001_installed
integration.states.test_pkg.PkgTest.test_pkg_002_installed_with_version
integration.states.test_pkg.PkgTest.test_pkg_003_installed_multipkg
integration.states.test_pkg.PkgTest.test_pkg_004_installed_multipkg_with_version
integration.states.test_pkg.PkgTest.test_pkg_011_latest
integration.states.test_pkg.PkgTest.test_pkg_013_installed_with_wildcard_version
integration.states.test_pkg.PkgTest.test_pkg_014_installed_missing_release
integration.modules.test_pkg.PkgModuleTest.test_install_remove
```
https://jenkins.saltstack.com/job/branch_tests/job/2017.7/view/Python%203/job/salt-linode-py3-fedora25/233/
https://jenkins.saltstack.com/job/branch_tests/job/2017.7/job/salt-linode-fedora26/62/

ping @terminalmage mind taking a look at these tests?

**Comment by Ch3LL** [comment]
ping @terminalmage now that the 2017.7.2 PR has been submitted can you take a look here now?

**Comment by terminalmage** [comment]
Yes, I'm looking today.

**Comment by terminalmage** [comment]
@Ch3LL https://github.com/saltstack/salt/pull/43360 should fix this, though the fixes needed to be made in 2016.11, so we'll need to wait for a merge-forward to see the branch builds stabilize.

**Comment by Ch3LL** [comment]
These are now passing. Thanks!!
